plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -2535,6 +2535,52 @@ void testDereferencePushdownWithNestedFieldsIncludingArrays()
         deleteIndex(tableName);
     }
 
+    @Test
+    public void testWildcardTableSameSchema()
+            throws IOException
+    {
+        String suffix = randomNameSuffix();
+        String firstIndex = format("test_wildcard_%s_1", suffix);
+        String secondIndex = format("test_wildcard_%s_2", suffix);
+        String wildcardTable = format("test_wildcard_%s_*", suffix);
+
+        String mappings =
+                """
+                {
+                    "properties": {
+                      "long_column":      { "type": "long" },
+                      "text_column":      { "type": "text" }
+                    }
+                }
+                """;
+
+        createIndex(firstIndex, mappings);
+        index(firstIndex, ImmutableMap.<String, Object>builder()
+                .put("long_column", 1L)
+                .put("text_column", "Trino")
+                .buildOrThrow());
+        createIndex(secondIndex, mappings);
+        index(secondIndex, ImmutableMap.<String, Object>builder()
+                .put("long_column", 2L)
+                .put("text_column", "rocks")
+                .buildOrThrow());
+
+        try {
+            assertQuery("DESCRIBE \"" + wildcardTable + "\"", "VALUES ('long_column', 'bigint', '', ''), ('text_column', 'varchar', '', '')");
+            assertQuery("SELECT * FROM \"" + wildcardTable + "\"", "VALUES (1, 'Trino'), (2, 'rocks')");
```

Comment:
Let's use `asserThat(query(...))` instead. `assertQuery` method isn't recommended. 
Same for `BaseOpenSearchConnectorTest`